### PR TITLE
refactor: make use of `GeneratedType`

### PR DESCRIPTION
**Pull request by xiduzo**
Replaced manual interface by their `GeneratedType` counterpart 

Solves [PZ-4776](https://dimpact.atlassian.net/browse/PZ-4776)

### CODE REVIEW

File: src/main/app/src/app/admin/zaakafhandel-parameters.service.ts [hunk 1]
```diff
@@ -64,9 +64,11 @@ export class ZaakafhandelParametersService {
       );
   }
 
-  listResultaattypes(zaaktypeUuid: string): Observable<Resultaattype[]> {
+  listResultaattypes(zaaktypeUuid: string) {
```
Comment (bas-info-nl) [inline src/main/app/src/app/admin/zaakafhandel-parameters.service.ts:67]: Why no explicit result type?

**Review by bas-info-nl — APPROVED** [review]

**Review by edgarvonk — APPROVED** [review]
nice!

File: src/main/app/src/app/admin/zaakafhandel-parameters.service.ts (hunk 1 above)
Comment (xiduzo) [reply src/main/app/src/app/admin/zaakafhandel-parameters.service.ts:67]: we do this already by setting the type explicitly on `.get<GeneratedType<"RestResultaattype">[]>`, not need to add double the explicity type

**Review by infonl-marcel — APPROVED** [review]
💪🏽

**Pull request merged**